src/beast/beast/nudb/detail/gentex.h

```diff
@@ -73,7 +73,7 @@ gentex_t<_>::lock()
 
 template <class _>
 void
-gentex_t<_>::unlock()
+gentex_t<_>::finish()
 {
     std::unique_lock<
```

Comment:
You really 
need two 
lines for 
the 40 
characters 
used to 
define this 
single variable?